ui/src/app/cohort-search/overview/overview.component.ts

```diff
@@ -24,16 +26,19 @@ export class OverviewComponent {
   @Input() chartData$: Observable<List<any>>;
   @Input() total$: Observable<number>;
   @Input() isRequesting$: Observable<boolean>;
+  @select(searchRequestError) error$: Observable<boolean>;
 
   cohortForm = new FormGroup({
     name: new FormControl('', [Validators.required]),
     description: new FormControl()
   });
 
+  error: boolean;
   stackChart = false;
   showGenderChart = true;
   showComboChart = true;
   showConflictError = false;
+  subscription: Subscription;
```

Comment:
Are we using this subscription anywhere in this component?

Reply:
Good catch. I ended up using the async pipe in the template instead. Removed